packages/apm/src/integrations/tracing.ts

```diff
@@ -182,6 +182,20 @@ export class Tracing implements Integration {
   public setupOnce(addGlobalEventProcessor: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
     Tracing._getCurrentHub = getCurrentHub;
 
+    if (global.performance) {
+      // The Performance object has a limited buffer size, often 150 entries. At some point the buffer may overflow, in
+      // which case we would not be able to use it to create/update spans.
+      // https://developer.mozilla.org/en-US/docs/Web/API/Performance
+      const oldCallback = performance.onresourcetimingbufferfull;
```

Comment:
Not available in IE, Opera and Safari iOS
https://developer.mozilla.org/en-US/docs/Web/API/Performance/onresourcetimingbufferfull

Reply:
Which is ok, right? We can only do best effort

Reply:
we need to check for existence, otherwise it breaks :)

Reply:
We do when we call it
![image](https://user-images.githubusercontent.com/363802/76501631-ea50bb80-6442-11ea-871c-1b15ed592cb4.png)
nothing breaks